app/src/main/java/com/dessalines/thumbkey/ui/components/settings/lookandfeel/LookAndFeelActivity.kt

```diff
@@ -750,6 +736,56 @@ fun LookAndFeelActivity(
                     onValueChange = { text = it },
                     label = { Text(stringResource(R.string.test_out_thumbkey)) },
                 )
+                if (showConfirmResetDialog.value) {
```

Comment:
Why is this inside of a random outlined text field, it should be at the top level.

Reply:
Ahh, think I misunderstood what you were saying.